mozilla/blurts-server
# MNTOR-2022 - add basic Cirrus integration

# References: 
Jira: MNTOR-2022




# Description

Add basic Cirrus integration:

1. derive user ID from server-side session object
2. send user ID to Cirrus, obtain enabled features
3. send user ID to client-side Glean JS component

# How to test

1. Load any page in the current production `(nextjs_migration)` set of pages
2. Observe in browser console and [Glean Debug Ping Viewer](https://debug-ping-preview.firebaseapp.com/pings/fx-monitor-local-dev) that pings are being sent
3. Verify that pings from unauthenticated sessions have a `user_id` that begins with `guest-`, and are different from authenticated sessions

# Checklist (Definition of Done)
- [x] Playwright needs access to new `NIMBUS_UUID_NAMESPACE` and `NIMBUS_SIDECAR_URL` env vars
